app/controllers/pets_controller.rb

```diff
@@ -31,6 +31,17 @@ def destroy
     redirect_to '/pets'
   end
 
+  def adopt_update
+    pet = Pet.find(params[:id])
+    if pet.adoption_status == "adoptable"
+      pet.adoption_status = "adoption pending"
+    elsif pet.adoption_status == "adoption pending"
+      pet.adoption_status = "adoptable"
+    end
+    pet.update({adoption_status: pet.adoption_status})
+    redirect_to "/pets/#{pet.id}"
+  end
```

Comment:
I know that this is probably too bulky for the controller, but I couldn't get it to work in the model and it is functionality from the individual project that needs to be present in order to build on it in this project. I propose adding it to the issues to come back to later if we have time.

Reply:
It wouldn't hurt to add it to the issues, even if we don't have time to come back to it.